.github/workflows/vertexai.yml

```diff
@@ -15,6 +15,21 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  spm-package-resolved:
+    runs-on: macos-14
+    outputs:
+      package_resolved_hash: ${{ steps.swift_package_resolve.outputs.package_resolved }}
+    env:
+      FIREBASECI_USE_LATEST_GOOGLEAPPMEASUREMENT: 1
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Should the cache be in this job instead?

When I run swift package resolve locally I see the following at the end of the log

```
Working copy of https://github.com/apple/swift-log.git resolved at 1.6.1
Creating working copy for https://github.com/google/GoogleAppMeasurement.git
Working copy of https://github.com/google/GoogleAppMeasurement.git resolved at 11.1.0
Creating working copy for https://github.com/apple/swift-nio-ssl.git
Working copy of https://github.com/apple/swift-nio-ssl.git resolved at 2.27.2
Creating working copy for https://github.com/google/app-check.git
Working copy of https://github.com/google/app-check.git resolved at 11.0.1
Fetching binary artifact https://dl.google.com/firebase/ios/swiftpm/11.1.0/FirebaseAnalytics.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/bin/firestore/11.2.0/rc0/FirebaseFirestoreInternal.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/bin/grpc/1.65.1/rc1/grpc.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/bin/grpc/1.65.1/rc1/grpcpp.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/bin/grpc/1.65.1/rc1/openssl_grpc.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/bin/abseil/1.2024011602.0/rc1/absl.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/swiftpm/11.1.0/GoogleAppMeasurement.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/swiftpm/11.1.0/GoogleAppMeasurementIdentitySupport.zip from cache
Fetching binary artifact https://dl.google.com/firebase/ios/swiftpm/11.0.0/rc1/GoogleAppMeasurementOnDeviceConversion.zip from cache
Fetched https://dl.google.com/firebase/ios/swiftpm/11.1.0/GoogleAppMeasurementIdentitySupport.zip from cache (0.30s)
Fetched https://dl.google.com/firebase/ios/swiftpm/11.0.0/rc1/GoogleAppMeasurementOnDeviceConversion.zip from cache (0.33s)
Fetched https://dl.google.com/firebase/ios/swiftpm/11.1.0/GoogleAppMeasurement.zip from cache (0.54s)
Fetched https://dl.google.com/firebase/ios/bin/abseil/1.2024011602.0/rc1/absl.zip from cache (1.68s)
Fetched https://dl.google.com/firebase/ios/swiftpm/11.1.0/FirebaseAnalytics.zip from cache (1.99s)
Fetched https://dl.google.com/firebase/ios/bin/firestore/11.2.0/rc0/FirebaseFirestoreInternal.zip from cache (2.91s)
Fetched https://dl.google.com/firebase/ios/bin/grpc/1.65.1/rc1/openssl_grpc.zip from cache (3.11s)
Fetched https://dl.google.com/firebase/ios/bin/grpc/1.65.1/rc1/grpcpp.zip from cache (8.04s)
Fetched https://dl.google.com/firebase/ios/bin/grpc/1.65.1/rc1/grpc.zip from cache (9.64s)